plugins/Goals/lang/en.json

```diff
@@ -68,30 +68,30 @@
         "Filename": "filename",
         "GoalConversion": "Goal conversion",
         "GoalConversions": "Goal conversions",
-        "GoalConversionsBy": "Goal %s conversions by type of visit",
-        "GoalIsTriggered": "Goal is triggered",
-        "GoalIsTriggeredWhen": "Goal is triggered when",
+        "GoalConversionsBy": "%s goal conversions by type of visit",
+        "GoalIsTriggered": "A goal is triggered",
+        "GoalIsTriggeredWhen": "A goal is triggered when",
         "GoalName": "Goal Name",
         "Goals": "Goals",
         "NGoals": "%s goals",
         "NRevenue": "%s revenue",
         "NItems": "%s items",
         "ManageGoals": "Manage Goals",
-        "GoalsOverview": "Goals Overview",
-        "GoalsOverviewDocumentation": "This is an overview of your goal conversions. Initially, the graph shows the sum of all conversions. %s Below the graph, you can see conversion reports for each of your goals. The sparklines can be enlarged by clicking on them.",
-        "GoalX": "Goal %s",
-        "HelpOneConversionPerVisit": "If a Page matching this Goal is refreshed or viewed more than once in a Visit, the Goal will only be tracked the first time the page was loaded during this visit.",
+        "GoalsOverview": "Overview og Goals",
+        "GoalsOverviewDocumentation": "An overview of your goal conversions. Initially, the graph shows the sum of all conversions. %s Below the graph, you can see conversion reports for each of your goals. The sparklines can be enlarged by clicking on them.",
+        "GoalX": "%s Goal",
+        "HelpOneConversionPerVisit": "If a Page matching this goal is refreshed or viewed more than once during a visit, the goal will only be tracked the first time the page was loaded during the visit.",
         "IsExactly": "is exactly %s",
-        "LearnMoreAboutGoalTrackingDocumentation": "Learn more about %1$s Tracking Goals in Matomo%2$s in the user documentation.",
-        "LeftInCart": "%s left in cart",
+        "LearnMoreAboutGoalTrackingDocumentation": "Learn more about tracking %1$s goals in Matomo%2$s in the user documentation.",
```

Comment:
?

Reply:
Note: The placeholders insert a link tag to https://matomo.org/docs/tracking-goals-web-analytics/
so might be fine to keep the `tracking` within the link...

Reply:
Ah, I thought it was a placeholder for something like "Tracking conversion/referral/campaign goals"